tests/cases/draw_rect_multiple_color_attachment.amber

```diff
@@ -36,5 +36,5 @@ PIPELINE graphics my_pipeline
 END
 
 RUN my_pipeline DRAW_RECT POS 0 0 SIZE 800 600
-EXPECT framebuffer IDX 0 0 SIZE 800 600 EQ_RGBA 0 128 0 204
+EXPECT framebuffer IDX 0 0 SIZE 800 600 EQ_RGBA 0 127 0 204
```

Comment:
Hm, I think this will fail on a vulkan host now (as it worked previously). Is this just a numerical difference between the dawn and vulkan engines?

Reply:
I see your point however right now Vulkan fails with 128 and passes with 127, at least on my machine. Is there any reasons for this to behave differently on different machines?

Reply:
Maybe? But, if both Vulkan and Dawn engines pass with this on your machine then I'm good with it. Maybe add a comment that we flipped the 128 to a 127 and if folks see this failing we need to find a better solution than changing the number? Just so we don't end up constantly flipping it back and forth, heh.

Reply:
How about changing 0.5 to 0.499 so eliminate the confusion?

Reply:
If that works, go for it.

Reply:
Yeah, I suspect something funny with quantization of the very-low-precision floating point used in rendering.  I could see it flipping back and forth. 

All the other AmberScript scripts in the tests/cases directory use 127, _except_ for the clear_color.amber script, but it's using an explicit 128 in that component of the clear color and it's using a UNORM so that's right in the middle of the range.

In short, I'm ok with this change. But let's keep an eye on this as an issue coming up.